components/mpas-ocean/driver/ECDA_mpaso_obs_mod.F

```diff
@@ -0,0 +1,636 @@
+!BOP
```

Comment:
@osculation please try follow similar format for comments in mpas-ocean files.  You can look at other files in the folders.

Also it would be best to stick with consistent naming of files too, e.g. mpas_ocn_ECDA_obs... etc.

Reply:
That is a good idea.